week-03/main.js

```diff
@@ -3,34 +3,56 @@
 // Given the following array
 var list = [1, 2, 3, 4, 5];
 // Remove one from the beginning of the array and add it to the end
-
-
+list.shift();
+list.push(1);
 // # Objects
 
 // Create an object with key value pairs for
 // `name` - a string,
 // `isAlive` - a boolean,
 // and `hobbies` an array.
 // Assign the object to the variable `user`
+var user = {name: 'margie', isAlive: true, hobbies:[]};
 
 
 
-// Add 'golf' to the `user`'s `hobby` array.
 
+// Add 'golf' to the `user`'s `hobby` array.
+user.hobbies = 'golf';
```

Comment:
This sets `hobbies` to a string. How would you __add__ "golf" to the hobbies array?